app/models/data_source.rb

```diff
@@ -136,7 +136,7 @@ def setup
     end
 
     def reload_source(clear_first = false)
-      logger.info { "Begin reload of data source #{id} for series #{self.series.name} [#{description}]" }
+      Rails.logger.info { "Begin reload of data source #{id} for series #{self.series.name} [#{description}]" }
```

Comment:
This is necessary?

Reply:
For both of these things I was grasping at straws.